bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java

```diff
@@ -2108,17 +2108,6 @@ MenuItem getMenuItem (int id) {
 	return null;
 }
 
-int getMessageCount () {
-	/*
-	 * On Windows10 (update 18272), an NPE is seen in below code which leads to a
-	 * possible crash, adding a null check for synchronizer instance. For more
-	 * details refer bug 540762
-	 */
-	if (synchronizer != null) return synchronizer.getMessageCount ();
```

Comment:
no, i will check the code. Thanks for the hint!!

Reply:
The former VtblCall in init () is now named DeleteList - just another name - so it could still happen:
The error was that the super constructor of Display which is Device is called before the initialization of the field synchronizer
( see https://stackoverflow.com/questions/23093470/java-order-of-initialization-and-instantiation). And it calls init() which is overwritten in Displays.init() - still before field initialization  of Display happens. I'll propose a solution tomorrow.